src/cts/src/SinkClustering.cpp

```diff
@@ -34,7 +34,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 #include "SinkClustering.h"
-#include "stt/SteinerTreeBuilder.h"
```

Comment:
I don't see any reason cts should be using pd instead of stt, so why don't you take the opportunity to correct it.